keyboards/kinesis/kint36/keymaps/kzar/keymap.c

```diff
@@ -0,0 +1,388 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Since I based this file originally on the [kemap.c file from Xyverz's map](https://github.com/qmk/qmk_firmware/blob/master/keyboards/kinesis/keymaps/xyverz/keymap.c) I'm wondering if it's appropriate to claim copyright for the file?

Reply:
I have added the header like you instructed, but also gave Xyverz credit. Let me know if you'd like me to adjust that.